demo_modules/_template/brick.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "template",
+    "client_path": "./template_client.ts",
+    "server_path": "./template_server.ts"
+  }
```

Comment:
add testonly so that this module doesn't get added to any collections by default.